src/components/atoms/review-content/review-content.tsx

```diff
@@ -0,0 +1,6 @@
+import { marked } from 'marked';
+import styles from './review-content.module.scss';
+
+export const ReviewContent = ({ content }: { content: string }): JSX.Element => (
+  <section className={styles['review-content']} dangerouslySetInnerHTML={{ __html: marked.parse(content) }}></section>
```

Comment:
Isnt the review content we get from the docmap already html?

Reply:
Correct! You passed the test! Your special prize is working with a forgetful idiot, Congratulations! 🤦 

Not sure why I had forgotten since I've been saying for a while we don't need marked anymore on the server. I just straight up lifted it as an example from server, and didn't question bringing in marked at all.

Anyway, pushed a simpler version now.